tests/conftest.py

```diff
@@ -327,9 +327,13 @@ def cmd(cmd, cwd=None, stderr=None, env=None):
     # stdout from cmd is captured and returned. The command is run in
     # a python subprocess so that program-level setup and teardown
     # happen fresh.
-    cmd = 'west ' + cmd
-    if not WINDOWS:
-        cmd = shlex.split(cmd)
+    if isinstance(cmd, str):
+        cmd = 'west ' + cmd
+        if not WINDOWS:
+            cmd = shlex.split(cmd)
```

Comment:
It's totally not your fault but using `shlex` here was a bad idea. Let me look it into it.

https://github.com/zephyrproject-rtos/west/pull/266#discussion_r284670076

Reply:
I have some working, should submit a PR that removes shlex today.

Reply:
Done, please take a look. It's long (and was tedious) but it's conceptually dead simple:
- #730

Reply:
Rebased this PR on top of #730

Reply:
#730 merged, please rebase